charts/api-server/templates/deployment.yaml

```diff
@@ -36,6 +36,8 @@ spec:
             value: "{{ .Values.mongoDSN }}"
           - name: POSTMANEXECUTOR_URI
             value: "{{ .Values.postmanExecutorURI }}"
+          - name: TESTKUBE_TELEMETRY_DISABLED
```

Comment:
Can we have naming with enabled instead of disabled, disabled = false can create some confusion.

Reply:
For me it doesn't matter honestly speaking. If TESTKUBE_TELEMETRY_DISABLED = false it means that it's enabled etc. Whatever you prefer. 
@jasmingacic , if we change this name you will have to change it in api-server code-base.

Reply:
yes i would have to change it in the code which hasn't been merged yet https://github.com/kubeshop/testkube/pull/475